docs/milestones.md

```diff
@@ -0,0 +1,40 @@
+## Milestones/Hitos
+
+### [Milestone 0] Modelo del problema
+Elaborar un modelo simple que sirva de base para describir los problemas de las Historias de Usuario (#2, #3, #4).
+
+PMV: Modelo incial del proyecto que atienda a las peticiones de #2, #3, #4 a un nivel básico, sobre un programa de organización de horarios.
+
+El modelo vendrá detallado con los elementos y sus atributos individuales empleados.
+
+Viabilidad: Presentar una lógica y elementos de un problema de forma que se pueda formular una solución para el problema presentado en las HUs #2, #3, #4 .
+
+Entidades encontradas en los HUs son implementadas en código fielmente.
+
+### [Milestone 1] Implementación lógica de negocio
+Necesito generar un nuevo horario/s partiendo de asignaturas, grupos, turnos y cursos concretos de un grado que sea factible, es decir que no haya ningún impedimento a la hora de asistir a una asginatura por coincidir con otra.
```

Comment:
Si ya sabes que tienes que usar "asignaturas, grupos, turnos y cursos" y no #2 #3 y #4 ¿para qué lo mencionas aquí y en el milestone anterior?

Reply:
Porque antes lo tenía señalado a la historia de usuario #2 como un ejemplo para que un tercero lo entienda, pero en otro comentario me dice que si solo estoy aplicando entonces un milestone a un único problema de un cliente concreto y considero que el milestone 1 es una función general para los tres aunque en los errores diga que "enlazar todas las historias de usuario sólo indica dejadez y falta de precisión" creo que con múltiples casos diferentes se puede estudiar mejor la función de crear un horario con información suelta.

Reply:
Por favor, lee de nuevo mi comentario. Si no tratáis de entender cuál es el problema es imposible que lo resolváis.
A ver si lo formulo de otra forma. ¿Te parece que haya una contradicción entre lo siguiente?
1. Milestone 0: "Elaborar un modelo simple"
2. Milestone 1: "partiendo de asignaturas, grupos, turnos y cursos concretos "

¿El modelo simple qué se hace con él entonces? ¿Se tira a la basura?

Reply:
https://github.com/ChinChainis/Proyecto_Reparahorarios_IV2425/commit/ab55d26b1557fe32e62d363dd7ddde41c808e824